packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -259,6 +259,29 @@ export default class ReconnectionManager {
         // Reconnect has failed
         LoggerProxy.logger.error('ReconnectionManager:index#reconnect --> Reconnection failed.', reconnectError.message);
         LoggerProxy.logger.info('ReconnectionManager:index#reconnect --> Sending reconnect abort metric.');
+
+
+        if (reconnectError instanceof NeedsRejoinError) {
+          // send call aborded event with catogery as expected as we are trying to rejoin
+          Metrics.postEvent({
+            event: eventType.CALL_ABORTED,
+            meeting: this.meeting,
+            data: {
+              errors: [
+                {
+                  category: errorObjects.category.expected,
```

Comment:
instead of duplicating the metrics call here, can you use a variable and assign the correct category instead?